src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -145,6 +147,9 @@ public NuGetPackage()
         /// </summary>
         protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
+            _nuGetPowerShellUsageCollector = new NuGetPowerShellUsageCollector();
+            NuGet.Common.TelemetryActivity.NuGetTelemetryService = new NuGetVSTelemetryService();
```

Comment:
@nkolev92 
Reason I did it was if I import Nuget.Common then just 8-9 lines down on `AsyncLazy<IVsMonitorSelection>` needed exact same thing it needs fully qualified name. So instead of touching existing code for same change I tried to contain my changes.
![image](https://user-images.githubusercontent.com/8766776/105566410-02d76680-5ce1-11eb-9448-9ece9a410257.png)

Reply:
You can always do a static import.

Reply:
Addressed in new [PR#3863](https://github.com/NuGet/NuGet.Client/pull/3863).